src/main/scala/org/antipathy/scoozie/action/SshAction.scala

```diff
@@ -1,13 +1,14 @@
 package org.antipathy.scoozie.action
 
-import scala.xml.Elem
-import org.antipathy.scoozie.configuration.Args
-import scala.collection.immutable._
 import com.typesafe.config.Config
-import scala.collection.JavaConverters._
-import com.typesafe.config.ConfigException
+import org.antipathy.scoozie.configuration.Args
 import org.antipathy.scoozie.exception.ConfigurationMissingException
 
+import scala.collection.JavaConverters._
+import scala.collection.immutable._
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Imports should be sorted alphabetically](https://app.codacy.com/app/SimonJPegg/scoozie/pullRequest?prid=3072461)